apps/spanishwotd/spanish_wotd.star

```diff
@@ -7,24 +7,48 @@ Author: logancornelius
 
 load("cache.star", "cache")
 load("encoding/json.star", "json")
-load("html.star", "html")
+load("re.star", "re")
 load("http.star", "http")
 load("render.star", "render")
 
-DEFAULT_WHO = "world"
-
-SPANISH_DICTIONARY_URL_1 = "https://www.dictionaryapi.com/api/v3/references/spanish/json/"
-SPANISH_DICTIONARY_URL_2 = "?key=d5ccb112-d698-45d6-ad07-c17a7b829b8d"
 CACHE_KEY = "wotd"
 CACHE_LLAVE = "pdd"
-CACHE_TTL = 10800  # 3 hours
-WOTD_CALENDAR_URL = "https://www.merriam-webster.com/word-of-the-day/calendar"  # Most succinct wotd definition
+CACHE_TTL = 3600  # 1 hour
```

Comment:
I had an issue where this app was not showing the current day's word, so I reduced this cache a bit to see if that would help.